integrationTest/providers.json

```diff
@@ -43,6 +43,12 @@
     "domain": "$DIGITALOCEAN_DOMAIN",
     "token": "$DIGITALOCEAN_TOKEN"
   },
+  "DNSME": {
```

Comment:
Can we use a name that is longer than "ME"?  (for this and the folder name)

Reply:
I can rename DNSME to DNSMADEEASY everywhere. It looked too long to me at first, but maybe it's ok?